src/service/extensions/tatumsdk.container.ts

```diff
@@ -0,0 +1,22 @@
+import { Constructable } from "typedi/types/types/constructable.type";
+import { AbstractConstructable } from "typedi/types/types/abstract-constructable.type";
+import { Token as DiToken } from "typedi/types/token.class";
+import { ContainerInstance, ServiceIdentifier } from "typedi";
+
+export interface ITatumSdkContainer {
+    get<T>(type: Constructable<T>): T;
+    get<T>(type: AbstractConstructable<T>): T;
+    get<T>(id: string): T;
+    get<T>(id: DiToken<T>): T;
```

Comment:
since idea was to have abstraction over typedi, maybe better not to use their types in the contained abstraction?

Reply:
That's a good point but we are using eg. Tokens inside so for extension to be able to retreive it they need to have it too...

Actually thinking about it now it's totally doable, give me a sec

Reply:
done